clients/python-wrapper/tests/integration/test_managers.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from tests.integration.conftest import _setup_repo, get_storage_namespace
+
+
+@pytest.fixture(name="setup_repo_with_branches_and_tags", scope="session")
+def fixture_setup_repo_with_branches_and_tags():
+    clt, repo = _setup_repo(get_storage_namespace("branches-and-tags"),
+                            "branches-and-tags",
+                            "main")
+    for i in range(10):
+        for j in range(100):
```

Comment:
A lot of branches and tags - we are just checking the API here

Reply:
Not too much :) to check the api we want to check edge cases and pagination so we need more than the default page size